PaddlePaddle/PaddleDetection
# Add model zoo unittest & wheel building script

**Add model zoo unittest & wheel building script**
- add unittests for `ppdet.model_zoo`
- add `scripts/build_wheel.sh` for wheel building and testing (build & install -> all unittests check)
- remove `ppdet/modeling/tests/test_transformer.py` for it compare diff between new OPs and old OPs, but old OPs are deleted
- remove `AnchorGenerator` unittest and `AnchorGeneratorRPN` for `ops.anchor_generator` deleted
- fix bugs in unittests

![屏幕快照 2021-05-18 上午10 14 25](https://user-images.githubusercontent.com/12605721/118580103-dda3c200-b7c1-11eb-964a-54a625ec9e89.png)


**Usage:**
```
cd PaddleDetection
bash scripts/build_wheel.sh
# get wheel in ./dist/paddledet-2.x.x-*.whl
```
